### ISSUE TITLE
The co mocha is a bit incompiatble with browser.

### ISSUE THREAD

**Issue by lygstate** [blakeembrey/co-mocha#19]
```
    Initialization
        should initialize the main object ‣

        "use strict";
                let contestData = {
                    question1: 'How Are you?',
                    answer1: 'Fine, thanks.',
                    question2: 'Status',
                    answer2: 'Great',
                    question3: 'Who is the greatest painter?',
                    answer3: 'Raphael'
                };
                expect(contestData.question1).to.equal('How Are you?');
                expect(GetSystemThemeName()).to.equal('aero');
                expect(testSystemInformation()).to.be.true;

        should do something ‣

        return co.call(this, fn.apply(this, arguments));
```

The result content of `should do something ‣` exposeed co.call`
We may need more specific function content.

**Comment by blakeembrey** [comment]
I'm not sure what you're asking/saying? Do you have more code examples or something? Where is `return co.call(this, fn.apply(this, arguments))` coming from?

**Comment by lygstate** [comment]
@blakeembrey  It's comes from the co-mocha,
Cause co-mocha wrappign the orignal testing function.
So the fn.toString() return `return co.call(this, fn.apply(this, arguments));`
But the original function content.

**Comment by blakeembrey** [comment]
Ok, I think I understand what you're having trouble with now. The language barrier made it extremely difficult to understand what issue you were having, since I already know the function is wrapped. I'll see if I can patch and release a new version tonight.

**Comment by blakeembrey** [comment]
Patched and released with `1.1.2` :+1:

**Comment by lygstate** [comment]
Great work, tested and verified.
